src/stories/PopoverMenu.stories.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {SafeAreaProvider} from 'react-native-safe-area-context';
+import PopoverMenu from '../components/PopoverMenu';
+import * as Expensicons from '../components/Icon/Expensicons';
+import MenuItem from '../components/MenuItem';
+import colors from '../styles/colors';
+
+/**
+ * We use the Component Story Format for writing stories. Follow the docs here:
+ *
+ * https://storybook.js.org/docs/react/writing-stories/introduction#component-story-format
+ */
+const story = {
+    title: 'Components/PopoverMenu',
+    component: PopoverMenu,
+};
+
+// eslint-disable-next-line react/jsx-props-no-spreading
+const Template = (args) => {
+    const [isVisible, setIsVisible] = React.useState(false);
+    const toggleVisibility = () => setIsVisible(!isVisible);
+    return (
+        <>
+            <MenuItem
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                {...args.menuItem}
+                onPress={toggleVisibility}
+                wrapperStyle={isVisible ? [{backgroundColor: colors.gray2}] : []}
+            />
+            <SafeAreaProvider>
+                <PopoverMenu
+                    // eslint-disable-next-line react/jsx-props-no-spreading
+                    {...args.popoverMenu}
+                    isVisible={isVisible}
+                    onClose={toggleVisibility}
+                    onItemSelected={toggleVisibility}
+                    menuItems={[
+                        {
+                            text: 'Bank account',
+                            icon: Expensicons.Bank,
+                            onSelected: toggleVisibility,
+                        },
+                        {
+                            text: 'Debit card',
+                            icon: Expensicons.CreditCard,
+                            onSelected: toggleVisibility,
+                        },
+                        {
+                            text: 'PayPal.me',
+                            icon: Expensicons.PayPal,
+                            onSelected: toggleVisibility,
+                        },
+                    ]}
+                />
+            </SafeAreaProvider>
+        </>
+    );
+};
+
+// Arguments can be passed to the component by binding
+// See: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Default = Template.bind({});
+Default.args = {
+    popoverMenu: {
+        anchorPosition: {
+            top: 80,
+            left: 20,
+        },
+    },
+    menuItem: {
+        title: 'Add payment Methods',
+        icon: Expensicons.Plus,
+    },
+};
+
+export default story;
+export {
+    Default,
+};
```

Comment:
I've not seen this before, would you mind explaining why we need to export `Default` on line 77?

Reply:
Hmm don't know the exact reason but this is how we do it on other stories too here's snippet from `MenuItem.stories.js`
https://github.com/Expensify/App/blob/d6e64b37e2cc38a692b0778de3348e6e7da924f0/src/stories/MenuItem.stories.js#L32-L34

Reply:
Let's find the reason then!

Reply:
> // Each story must be exported with a named export